hack/release.sh

```diff
@@ -12,7 +12,7 @@ echo "$REGISTRY_PASSWORD" | ko login -u "$REGISTRY_USERNAME" --password-stdin "$
 echo "Building container image"
 
 # Using defaults, this pushes to:
-# quay.io/shipwright/shipwright-operator/github.com/shipwright-io/build/cmd/manager:latest
-KO_DOCKER_REPO="$IMAGE_HOST/$IMAGE" GOFLAGS="${GO_FLAGS}" ko resolve -t "$TAG" -P -R -f deploy/ > release.yaml
+# quay.io/shipwright/shipwright-operator/manager:latest
```

Comment:
I don't think quay will like this, either :disappointed:. It only supports `quay.io/<org-name>/<repo-name>:<tag>`

Reply:
Yeah, I figured as much 😢 

I don't know how much the name of the image really matters to you all, now that developers aren't building+pushing themselves you might never notice what it's called.

The options outlined above give you two options basically, `quay.io/shipwright/shipwright-operator` (same as today, with `--bare`) or `quay.io/shipwright/manager` (with `-B` and `KO_DOCKER_REPO=quay.io/shipwright`).

Reply:
Decided to go with `--bare` and publish to `quay.io/shipwright/shipwright-operator:latest`